editor/src/components/canvas/canvas-strategies/absolute-resize-bounding-box-strategy.spec.tsx

```diff
@@ -48,754 +57,1177 @@ function multiselectResizeElements(
+        edgePosition: { x: 0.5, y: 1 } as EdgePosition,
+        drag: testDrag,
+        modifiers: shiftModifier,
+        bounding: testBounding,
+        draggedBounding: { left: 40, top: 50, width: 271, height: 325 },
+      },
+    ],
+    [
+      'right side, aspect ratio locked',
+      {
+        edgePosition: { x: 1, y: 0.5 } as EdgePosition,
+        drag: testDrag,
+        modifiers: shiftModifier,
+        bounding: testBounding,
+        draggedBounding: { left: 50, top: 41, width: 265, height: 318 },
+      },
+    ],
+    [
+      'top left corner, center mode, aspect ratio locked',
+      {
+        edgePosition: { x: 0, y: 0 } as EdgePosition,
+        drag: testDrag,
+        modifiers: altShiftModifier,
+        bounding: testBounding,
+        draggedBounding: { left: 65, top: 68, width: 220, height: 264 },
+      },
+    ],
+    [
+      'bottom left corner, center mode, aspect ratio locked',
+      {
+        edgePosition: { x: 0, y: 1 } as EdgePosition,
+        drag: testDrag,
+        modifiers: altShiftModifier,
+        bounding: testBounding,
+        draggedBounding: { left: 29, top: 25, width: 292, height: 350 },
+      },
+    ],
+    [
+      'top right corner, center mode, aspect ratio locked',
+      {
+        edgePosition: { x: 1, y: 0 } as EdgePosition,
+        drag: testDrag,
+        modifiers: altShiftModifier,
+        bounding: testBounding,
+        draggedBounding: { left: 35, top: 32, width: 280, height: 336 },
+      },
+    ],
+    [
+      'bottom right corner, center mode, aspect ratio locked',
+      {
+        edgePosition: { x: 1, y: 1 } as EdgePosition,
+        drag: testDrag,
+        modifiers: altShiftModifier,
+        bounding: testBounding,
+        draggedBounding: { left: 29, top: 25, width: 292, height: 350 },
+      },
+    ],
+    [
+      'top side, center mode, aspect ratio locked',
+      {
+        edgePosition: { x: 0.5, y: 0 } as EdgePosition,
+        drag: testDrag,
+        modifiers: altShiftModifier,
+        bounding: testBounding,
+        draggedBounding: { left: 71, top: 75, width: 208, height: 250 },
+      },
+    ],
+    [
+      'left side, center mode, aspect ratio locked',
+      {
+        edgePosition: { x: 0, y: 0.5 } as EdgePosition,
+        drag: testDrag,
+        modifiers: altShiftModifier,
+        bounding: testBounding,
+        draggedBounding: { left: 65, top: 68, width: 220, height: 264 },
+      },
+    ],
+    [
+      'bottom side, center mode, aspect ratio locked',
+      {
+        edgePosition: { x: 0.5, y: 1 } as EdgePosition,
+        drag: testDrag,
+        modifiers: altShiftModifier,
+        bounding: testBounding,
+        draggedBounding: { left: 29, top: 25, width: 292, height: 350 },
+      },
+    ],
+    [
+      'right side, center mode, aspect ratio locked',
+      {
+        edgePosition: { x: 1, y: 0.5 } as EdgePosition,
+        drag: testDrag,
+        modifiers: altShiftModifier,
+        bounding: testBounding,
```

Comment:
I think it would be better if the testBounding rectangle would be here inline, I forgot the rectangle while I scrolled here, and now I have to scroll back a lot to find the rectangle size, memorize it, then scroll back here (find the test again) and then compare it with the result :)

Reply:
I'd also like to suggest renaming `draggedBounding` to `expectedBounding`. When first reading I _thought_ it was the expected result, but had to then read through the code itself to find where it was used to verify that.

Reply:
Ok, actually that was my first version, I just wanted to eliminate redundancy, but I agree with your points, I revert to inline literals.